src/SyntaxHighlighter/include/SyntaxHighlighter/Cpp.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYNTAX_HIGHLIGHTER_CPP_H_
+#define SYNTAX_HIGHLIGHTER_CPP_H_
+
+#include <absl/container/flat_hash_map.h>
```

Comment:
This header is not needed here. (Once more the same problem is in the assembly highlighter. I really should review my own code. :see_no_evil:  My bad. )